images.html

```diff
@@ -0,0 +1,129 @@
+        <a class="link white" href="./index.html">Home</a>
+        <a class="link white" href="./Introduction.html">Components</a>
+      </nav>
+    </header>
+    <div class="main-content flex-row">
+      <aside class="sidebar">
+        <h3 class="sidebar-heading">Components</h3>
+        <ul class="unstyled">
+          <li><a href="./Introduction.html">Intro</a></li>
+          <li><a href="./Avatar.html">Avatar</a></li>
+          <li><a href="./alert.html">Alerts</a></li>
+          <li><a href="./badges.html">Badges</a></li>
+          <li><a href="./button.html">Button</a></li>
+          <li><a href="./cards.html">Cards</a></li>
+          <li><a href="./images.html">Images</a></li>
+          <li><a href="./input.html">Input</a></li>
+        </ul>
+      </aside>
+      <hr class="vertical-line" />
+      <main class="grow-1">
+        <h2 class="heading-1">Images</h2>
+        <p>
+          Images are an integral part of the content strategy of any website.
+          The best images are set above the fold and act as both eye candy and
+          informational tools. They should relate directly to the text of the
+          page, and ideally, they should be original pictures. Stay away from
+          stock clichés as much as you can.
+        </p>
+        <h4 class="heading-3">Example of Responsive Image</h4>
+        <div class="container-box m-radius">
+          <img
+            class="cover-img l-radius"
+            src="https://picsum.photos/200"
+            srcset="
+              https://picsum.photos/400 400w,
+              https://picsum.photos/600 600w
+            "
+            sizes="(min-width: 650px) 200px,
+                100px"
+          />
+        </div>
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=446.25&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=28px&ph=33px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%250A%2520%2520%2520%2520class%253D%2522cover-img%2520l-radius%2522%250A%2520%2520%2520%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%2522%250A%2520%2520%2520%2520srcset%253D%2522%250A%2520%2520%2520%2520%2520%2520https%253A%252F%252Fpicsum.photos%252F400%2520400w%252C%250A%2520%2520%2520%2520%2520%2520https%253A%252F%252Fpicsum.photos%252F600%2520600w%250A%2520%2520%2520%2520%2520%2520%2522%250A%2520%2520%2520%2520sizes%253D%2522%28min-width%253A%2520650px%29%2520200px%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520100px%2522%250A%252F%253E"
+          style="
+            width: 32rem;
+            height: 20rem;
+            border: 0;
+            transform: scale(1);
+            overflow: hidden;
+          "
+          sandbox="allow-scripts allow-same-origin"
+        >
+        </iframe>
+        <h4 class="heading-3">Example of Round Image</h4>
+        <div class="container-box m-radius">
+          <img
+            class="rounded-circle cover-img"
```

Comment:
You are using `l-radius`  at the top here it is `rounded-circle` .You should be consistent with class names.

Reply:
Ok noted. Previously when I have used libraries in other projects, I saw this pattern and it kind of remains in my head. That's why used m-radius, l-radius and for complete circle, I have used rounded-circle.